book/logging.md

```diff
@@ -0,0 +1,70 @@
+# RHTAP Logs
+
+Logging is crucial for effective monitoring and troubleshooting.\
+In the RHTAP environments, numerous clusters, components and services work together, making it essential to have comprehensive logs. Logging provides valuable insights into their behavior, allowing to understand and diagnose issues. Logging facilitates proactive monitoring, enabling prompt detection of failures, abnormal behavior, or potential security threats.\
```

Comment:
Obviously I read the previous sentence, still it is not clear. 
does "logging" provide insights to... logs behavior? or to <clusters, components, services>? Instead of starting a new sentence about logging, you can mention "those comprehensive logs provides"...

Reply:
I agree that it's not clear who's their.

Reply:
I merged those two sentences into one. Let me know if it's more clear now.

Reply:
LGTM